test/tar/tar_test.go

```diff
@@ -0,0 +1,65 @@
+package tar
```

Comment:
Thanks for adding unit test tar writer, but the placement of this test migh need to be updated.

Most of unit tests are located directly in the package they test (like auth), so Makefile test target [0] differentiates unit and other tests based on the package.  I'd suggest move this directory content into top-level tar directory (or update Makefile, as you prefer) to ensure the test is executed by make test.

[0] https://github.com/konveyor/tackle2-hub/blob/main/Makefile#L126-L128